.github/workflows/ci.yml

```
@@ -34,6 +34,17 @@ jobs:
     - name: Run test coverage
       if: matrix.node-version == '20.x'
       run: npm run test:coverage
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)